code/modules/vehicles/lavaboat.dm

```diff
@@ -70,4 +71,5 @@
 	D.vehicle_move_delay = 1
 	D.set_riding_offsets(RIDING_OFFSET_ALL, list(TEXT_NORTH = list(1, 2), TEXT_SOUTH = list(1, 2), TEXT_EAST = list(1, 2), TEXT_WEST = list( 1, 2)))
 	D.keytype = null
-	AddComponent(/datum/component/fishingbonus,80)
+	var/datum/component/fishingbonus/fb = LoadComponent(/datum/component/fishingbonus)
```

Comment:
not the correct way to use this component

Reply:
It is and it worked

Reply:
This is not the way you use components. Do not modify components like this.

Reply:
ok